vms/vm-under-test/scripts/customize-vm

```diff
@@ -90,21 +90,24 @@ cat <<'EOF' > "$checkup_boot_script_full_path"
 # Copyright 2024 Red Hat, Inc.
 #
 
-set -e
+set -x
 
 checkup_boot_ready_marker_full_path=$1
```

Comment:
This variable is set but not used.

Reply:
Fixed in the first commit, later it is removed.